packages/flutter/test/material/date_picker_test.dart

```diff
@@ -25,33 +25,35 @@ void main() {
     DateTime _selectedDate = new DateTime(2016, DateTime.JULY, 26);
 
     await tester.pumpWidget(
-      new Overlay(
-        initialEntries: <OverlayEntry>[
-          new OverlayEntry(
-            builder: (BuildContext context) => new StatefulBuilder(
-              builder: (BuildContext context, StateSetter setState) {
-                return new Positioned(
-                  width: 400.0,
-                  child: new SingleChildScrollView(
-                    child: new Material(
-                      child: new MonthPicker(
-                        firstDate: new DateTime(0),
-                        lastDate: new DateTime(9999),
-                        key: _datePickerKey,
-                        selectedDate: _selectedDate,
-                        onChanged: (DateTime value) {
-                          setState(() {
-                            _selectedDate = value;
-                          });
-                        },
+      new MaterialApp(
+        home: new Overlay(
```

Comment:
We need a MaterialApp (now) because DatePicker depends on localized tooltip strings. I've removed the Overlay stuff and simplified the test a bit by leaving out the date digit absolute coordinates.